app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] A post belongs to an `author`, not a `user`, So you should have this instead:

  ```
  belongs_to :author, class_name: 'User'
  ```

- [ ] I can also see from your database schema that what you have is `user_id` in the posts table. You need to update your references to point to the `author`, instead of the `user`. Refer to this comment for an idea of how to do that: [https://stackoverflow.com/a/42056089](https://stackoverflow.com/a/42056089)